webpack/ForemanInventoryUpload/Components/PageHeader/components/PageDescription/PageDescription.js

```diff
@@ -53,6 +54,16 @@ export const PageDescription = () => {
           />
         </Text>
       )}
+      <Text ouiaId="text-minimal-data-collection">
+        {__('Learn more about ')}
+        <a
+          href={getDocsURL('Managing_Hosts', 'setting-minimal-data-collection')}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {__('Minimal data collection setting')}
+        </a>
```

Comment:
This will not be properly translated. Needs a `<FormattedMessage>`

Reply:
@lfu did you want to make a follow up pr?